### DISCUSSION TITLE
URL link based one-time Meetings like Zoom or Google Meet and others offer

### DISCUSSION THREAD

**Discussion by jittygitty** [element-hq/element-meta#907] [Ideas]
#### What would you like to do?
Provide people a URL meeting link to a matrix hosted conference meeting that is somewhat anonymous and not tied down to actual "user-IDs" but people just pick their Username for the session when they join the meeting. This way you just send out a URL link to whoever you want to join your audio/video meeting, similar to ZOOM url links etc and the recipient doesn't even need to be registered anywhere.

Is this already doable and I missed how?

#### Why would you like to do it?
Would make a very quick way to host meetings, ideally with option to obfuscate registered user names from the invited conference participants. I noticed in Zoom etc one can simply type in the username/nickname for that particular session etc.

#### How would you like to achieve it?
Perhaps by hooking it up with Element Call? Or Jitsi etc?

### Have you considered any alternatives?

Current alternative is to user something else other than matrix powered conference.

### Additional context

I think such a feature can be a very powerful way to promote matrix to many more people and make it a popular alternative to proprietary zoom/teams/meet/others etc.

**Comment by turt2live** [comment]
This is essentially what happens already when you visit call.element.io or meet.element.io ?

**Comment by jittygitty** [comment]
@turt2live Thanks but is there a way we can create such links from our Android clients or at least to "Join" the links/meetings with Android clients instead of through a web browser?

On the Desktop/Windows for example, the links for Zoom/WebEx/Teams/etc will start a full Desktop "Application" which runs better and usually with more features than using it inside the "web browser". 

On the plus side, i hadn't realized we could use element call directly without logging in etc. But would be nice if we could create meet links or join meeting inside android or desk app.

UPDATE: Also in case it helps, a couple of the features that had issues in browser but worked well with full "app/software" with the competing systems were "remote control" where users can share view and mouse/keyboard control of their desktop and recording.
